plugins/rich-editor/src/scripts/Editor/Generic/Toolbar.tsx

```diff
@@ -237,11 +194,11 @@ export class Toolbar extends React.PureComponent {
     /**
      * Handle the simple on/off inline formats (eg. bold, italic).
      *
-     * @param {string} itemKey - The key of the item.
-     * @param {Object} itemData - The item to modify.
-     * @param {Object} range - The range to update.
+     * @param itemKey - The key of the item.
```

Comment:
Shouldn't these params be refactored to typescript format?

Reply:
Notice the types have been removed. I left text comments for the parameters, which is still valid here. We should definitely not keep the type annotations in the comments though. Parameter comments are optional, but in some cases nice to have.